### ISSUE TITLE
Editor in Role table in Admin interface

### ISSUE THREAD

**Issue by jochenklar** [rdmorganiser/rdmo#942]
### Description / Beschreibung

On `/admin/accounts/role/` "all Sites" is shown for "EDITORS" even when the user is only editor of one site.

**Comment by jochenklar** [comment]
... it actually works as expected, @MyPyDavid do you have any idea why I filed this issue?

**Comment by MyPyDavid** [comment]
I just checked and I don't get the "all Sites" at all, just a list of each site..

In `rdmo/accounts/admin.py` , the `getattr(obj, f'{field_name}__count', 0) ` counts 2x"all Sites" .
Should be fixed with adding `distinct=True` to the `Count`:
```
Count('member', distinct=True), Count('manager', distinct=True), Count('editor', distinct=True), Count('reviewer', distinct=True),
```

**Comment by jochenklar** [comment]
Ah I just added the change to #1014.

**Comment by MyPyDavid** [comment]
yeah, I saw #1014 only afterwards 🙈  , closed it.

**Comment by jochenklar** [comment]
Will be released in RDMO 2.2.
